### PR TITLE
Overview Template

### PR DESCRIPTION
**Pull request by wesrice**
Overview

### CODE REVIEW

**Review by kevcenteno — CHANGES_REQUESTED** [review]
This is quite the PR.  Lots of ideas were implemented here.  I have just a few change requests.

In general, how do you determine which components should have an a11y test (`<Overview>`) and which should not (`<LinkGroup[*]>`)?

File: content/geoip2/legacy/_alert.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Alert type="error">
```
[the remark this answers is not in the document]
Comment (wesrice) [reply content/geoip2/legacy/_alert.mdx:1]: So Gatsby will ignore the file and prevent it from making a page of it. This is custom functionality, but I used the same convention as scss.

https://github.com/maxmind/dev-site/pull/69/files#diff-24c29bd216c07004f4714e6285d1569e3ccf831b89ca9e49b94d05ac8682592eR26

**Comment by wesrice** [comment]
@kevcenteno I've made the requested changes.

I'm still figuring out when to add a11y tests. Generally, if a component has state changes, I try to add a test to that component. Otherwise, I've just been adding a11y to some to the broader components that integrates a lot of the smaller ones, like the overview template component.

File: content/minfraud/index.mdx [hunk 2]
```diff
@@ -1,17 +1,76 @@
...
+<Alert type="info">
   If you are a minFraud Legacy customer, please refer to our
   [minFraud Legacy documentation](/legacy/minfraud).
 
   If you are Proxy Detection Legacy customer, please refer to our
   [Proxy Detection Legacy documentation](/legacy/proxy-detection).
-
 </Alert>
+
+{/*<!-- lint disable code-block-style -->*/}
```
[the remark this answers is not in the document]
Comment (wesrice) [reply content/minfraud/index.mdx]: I disabled it. Remark lint was confusing indention within react components as indented markdown code blocks.

**Review by kevcenteno — APPROVED** [review]

File: content/minfraud/index.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kevcenteno) [reply content/minfraud/index.mdx]: I see. It looks like the `fix:content` is also broken; it's adding fence blocks to some of the `<LinkGroup>` occurrences in the mdx files

**Pull request merged**